source/Handlebars/Handlebars.csproj

```diff
@@ -66,9 +66,24 @@
     <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)'=='net5'">
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)'=='net6'">
+    <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+  </ItemGroup>
+
```

Comment:
Microsoft.CSharp is a polyfill package hence this dependency should be able to be removed on the newer frameworks.